app/src/main/java/ch/epfl/sdp/Tools.java

```diff
@@ -1,25 +1,33 @@
 package ch.epfl.sdp;
 
 import android.Manifest;
+import android.app.Activity;
 import android.content.Context;
 import android.content.pm.PackageManager;
-import android.os.Build;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
 import android.util.Log;
-import android.widget.Toast;
+import android.util.Pair;
 
-import androidx.annotation.NonNull;
 import androidx.biometric.BiometricManager;
-import androidx.biometric.BiometricPrompt;
 import androidx.core.app.ActivityCompat;
 
-import java.lang.reflect.Method;
-import java.util.concurrent.Executor;
+import com.google.firebase.firestore.GeoPoint;
 
-import static com.mapbox.mapboxsdk.Mapbox.getApplicationContext;
-
-public class BiometricUtils {
+/**
+ * This class includes some utilities used when dealing with biometric authentication
+ */
+public class Tools {
```

Comment:
It's great that you moved the tools related to network status here! Makes more sense indeed